### PR TITLE
Removing the Volumes for these images [ticket: DEVOPS-620]

### PR DESCRIPTION
**Pull request by goosemo**
Making what we want to happen be in this repo

### CODE REVIEW

File: 9.1/Dockerfile [hunk 1]
```diff
@@ -49,7 +49,6 @@ RUN mkdir -p /var/run/postgresql && chown -R postgres /var/run/postgresql
 
 ENV PATH /usr/lib/postgresql/$PG_MAJOR/bin:$PATH
 ENV PGDATA /var/lib/postgresql/data
-VOLUME /var/lib/postgresql/data
```
Comment (dblackdblack) [inline 9.1/Dockerfile:52]: I would replace `VOLUME /var/lib/postgresql/data` with `RUN mkdir -p /var/lib/postgresql/data`.  The `VOLUME` command also has the side-effect of creating the mount point directory if it doesn't already exist. I don't know for sure if this Dockerfile doesn't already create the directory, but `mkdir -p` is a no-op if the directory already exists, so it doesn't hurt to add it and I've been bitten by that side-effect in the past

Comment (goosemo) [inline 9.1/Dockerfile:52]: The entry point does this: https://github.com/plangrid/postgres/blob/master/docker-entrypoint.sh#L9

**Comment by goosemo** [comment]
I'm also not sure we should merge this? As it'll make it simpler for us to pull in new changes later. If I make this the default branch would anyone have a problem there?

**Comment by dblackdblack** [comment]
It will be really confusing if the docker image that's pushed up to the registry is from a Dockerfile in a feature branch

**Comment by goosemo** [comment]
What's your reasoning there? If I make this branch the default, when people clone it, that's where it's built from. Also humans wouldn't be making/pushing these ideally, but we don't have that yet. I'm mostly trying to make this as simple to maintain and keep updated with changes from the source.

**Pull request merged**